docs/database/schema-alternative.sql

```diff
@@ -0,0 +1,101 @@
+--
```

Comment:
@rhuss Thank you for this alternative. I played with the separate connector_property and action_property at first and then changed it to the common table, with relations to action and connector. Looking at both options again, I think I prefer the separate tables as you have done, albeit that means we have two tables with identical column definitions, which is normally one thing I hate...

My only question is about the action_instance table and how this relates to an Camel connector configuration for an integration, correlating the two together somehow could be trickier if we have multiple sources for the same data - how do we handle them getting out of sync? Which one is authoritative?